### DISCUSSION TITLE
Authenticated requests

### DISCUSSION THREAD

**Discussion by mmintel** [nextauthjs/next-auth#2365] [Help]
Hi there,

I am playing around with my API and wanna use Postman to make things easier. I've got my CSRF token from my user session and tried to set it as a header in my postman requests like so:

```
X-CSRF-TOKEN: 6750c6028fa3d35d0279237f9ce196c56c2acc994703baeecf3ddc33bf6640f9
```

but I am not authorized, no session. How can I do that?

**Comment by hems** [comment]
did you solve this in the end? ( :
